Gaijinizer patch/Map162.txt

```diff
@@ -330,10 +334,10 @@ It was time to check her USB memory.#MTLed
  メール転送用のデータがあった……
 > CONTEXT: Map162/events/2/pages/1/79/Dialogue/yuuma
 > CONTEXT: Map162/events/2/pages/2/65/Dialogue/yuuma
-Among them is my record from college ...
-In particular, the situation after entering the
-military was described in detail
-There was data for mail forwarding ...#MTLed
+On there were records about me from our college
+days. In particular, there was a very detailed
+forwarded email about me on what had happened
+after I joined the military...#RonIffy
 > END STRING
```

Comment:
Minor flow:
>_**On it were records about me from when I was in
college...**_
>
>
>_**In particular, there was a very detailed email
that had been forwarded to someone about what had
happened after I joined the military...**_